api/user/repository/userRepository.js

```diff
@@ -0,0 +1,34 @@
+import Sequelize from 'sequelize';
+const {DataTypes}=Sequelize;
+import db from 'dotenv';
+db.config();
+
+const sequelize = new Sequelize( {
+  host: process.env.HOST,
```

Comment:
move process.env.X  to the config/index.js file.
It is common practice to call process.env.X only in the config/index.js file.
In all other places, you only need to export specific keys from the config object in config/index.js